classes/01class/exercises/c01-git02/mail2kvanitha/README.md

```diff
@@ -0,0 +1,24 @@
+# c01-git02
```

Comment:
This file should not be part of the c01-git03 exercise delivery

Reply:
removed c01-git02 file from c01-git03 branch